pkgs/applications/science/math/wolfram/default.nix

```diff
@@ -59,13 +59,22 @@ let versions = callPackage ./versions.nix { };
 in
 
 callPackage ./generic.nix {
+
```

Comment:
Unnecessary empty lines.

Reply:
These empty lines act as visual helpers to increase the readability of the code. If the definitions are more than a few and some of them are longer than one line, I find the code with empty lines easier to read because the brain does not need to track the boundaries of the members, reducing the mental burden subconsciously. Compare the two snippets below:

![image](https://github.com/user-attachments/assets/dd56e494-78d5-48c9-bd22-f6c451b7fb23)

vesus

![image](https://github.com/user-attachments/assets/3ef76e0b-284e-42a1-a1c4-ccb9c2d695d9)

The one on the top seems too dense. This is a personal taste, of course.